cloudmanager/resource_netapp_cloudmanager_connector_gcp.go

```diff
@@ -413,3 +388,20 @@ func resourceOCCMGCPUpdate(d *schema.ResourceData, meta interface{}) error {
 	}
 	return resourceOCCMGCPRead(d, meta)
 }
+
+func getGCPServiceAccountKey(d *schema.ResourceData) (string, error) {
+	service_account_path := d.Get("service_account_path").(string)
```

Comment:
The convention in Go is to use mixedCaps rather than underscores to write multi-word names. So service_account_path should be serviceAccountPath. Same as serviceAccountKey.